src/composables/sdk.ts

```diff
@@ -0,0 +1,66 @@
+import { ref } from 'vue';
+import type {
+  IDefaultComposableOptions,
+  Protocol,
+} from '@/types';
+import { BaseProtocol } from '@/protocols/BaseProtocol';
+import { AeternityProtocol } from '@/protocols/aeternity/AeternityProtocol';
+import { BitcoinProtocol } from '@/protocols/bitcoin/BitcoinProtocol';
+import { DogecoinProtocol } from '@/protocols/dogecoin/DogecoinProtocol';
+import {
+  PROTOCOL_AETERNITY,
+  PROTOCOL_BITCOIN,
+  PROTOCOL_DOGECOIN,
+} from '@/popup/utils';
+import { watchUntilTruthy } from '@/utils';
+import { useAccounts } from './accounts';
+
+const isSdkRegistryReady = ref(true);
```

Comment:
If it's not returned by the composable it doesn't need to be reactive - use `let` instead.

Reply:
Removed it.